tests/python/test_genotype_extractor_worker.py

```diff
@@ -0,0 +1,144 @@
+import ast
+import mock
+import nose
+import nose.tools as asserts
+import json
+import vcf as pyvcf
+
+from cycledash import app, db
+from common.helpers import tables, find
+
+from test_projects_api import create_project_with_name
+from test_bams_api import create_bam_with_name
+from test_runs_api import create_run_with_uri
+
+from workers.genotype_extractor import _extract
+
+
+class TestGenotypeExtractorWorker(object):
+    def setUp(self):
+        self.project = create_project_with_name('my project')
+        self.run = create_run_with_uri(self.project['id'], '/tests/data/snv.vcf')
+
+    def tearDown(self):
+        with tables(db.engine, 'projects', 'vcfs', 'genotypes') as (con, projects, runs, genotypes):
```

Comment:
Since you're doing this in multiple tests, maybe worth a helper to `delete_tables(genotypes, runs, projects)`?